### DISCUSSION TITLE
Managing Grafana Alert Labels

### DISCUSSION THREAD

**Discussion by ryeleo** [grafana/grafana#60831] [Alerting]
It would be nice to have some means for managing Grafana's Alert Labels, e.g. in the Web GUI and/or via Grafana's provisioning folder solution.

### Scenario

I've been planning how we'll use Grafana Labels for the Route Views project. In this process, I've created several 'test' labels while learning about Unified Alerting. Also, I've accidentally had typos in many of my label values! So, at this moment I'd love to be able to *'clean up the test/typo labels.'* 

As a Grafana Administrator, I'd love to cleanup my Alert Labels so that:
- Only *standard labels* are suggested, and
- Only *standard values* are suggested for those labels in Grafana's Web GUI.

> Concretely, I'm trying to establish the following *standard labels*, with *standard values*:
> 
> * **status:** Status of this alert.
>     * draft: Designing/testing a new alert rule.
>     * production: This alert is up and running! 
>     > Notifications will be sent to our primary Slack channel.
> * **exposure:** How many of our Users are affected?
>     * low: Users may not even notice.
>     * med: Some users *may* notice.
>     * high: Some users *will* notice.
>     * urgent: Most of our users are affected.
> * **severity:** How much of an issue is this in our infrastructure?
>     * warn: Nothing is broken, *yet* ➡ Should probably look at this when there is free time.
>     * error: Something is broken ➡ Should look into during working hours.
>     * urgent: Something major is broken ➡ Drop everything and fix this!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
